experimental/handoff-library/csharp_dotnetcore/samples/LPProxyBot/LiverPersonConnector/LivePersonConnector.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace LivePersonConnector
+{
+    public interface ICredentialsProvider
+    {
+        string LpAccount { get; }
+        string LpAppId { get; }
+        string LpAppSecret { get; }
+        string MsAppId { get; }
+    }
+
+    internal class LivePersonConversationRecord
+    {
+        public string ConversationId;
+        public string MsgDomain;
+        public string AppJWT;
+        public string ConsumerJWS;
+    }
+
+    static class LivePersonConnector
+    {
+        static public async Task<LivePersonConversationRecord> EscalateToAgent(ITurnContext turnContext, IEventActivity handoffEvent, string account, string clientId, string clientSecret, ConversationMap conversationMap)
+        {
+            var sentinelDomain = await GetDomain(account, "sentinel");
+            var appJWT = await GetAppJWT(account, sentinelDomain, clientId, clientSecret);
```

Comment:
consider adding Async at the end of the asybc method. GetAppJwtAsync?

Reply:
Agreed